repobuilder/job.go

```diff
@@ -129,13 +141,28 @@ func (j *Job) linkPackages(dest string) error {
 		mirror := filepath.Join(dest, filepath.Base(pkg))
 		if j.release.IsDevelopmentBuild() {
 			if _, err := os.Stat(mirror); os.IsNotExist(err) {
-				err = errors.Wrap(os.Remove(mirror), "problem removing previous development build")
-				grip.Notice(err)
+				grip.Warning(message.WrapError(os.Remove(mirror), message.Fields{
```

Comment:
Why does this attempt to remove the directory if the directory already doesn't exist?

Reply:
it's always been this way. Will delete